react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.test.js

```diff
@@ -63,6 +63,15 @@ describe('SprkCheckboxItem:', () => {
     ).toBe(true);
   });
 
+  it('should apply visibility-toggle class', () => {
```

Comment:
Lets add one to make sure it is not applying it when `isVisibilityToggle={false}`